tox.ini

```diff
@@ -9,17 +9,18 @@ deps = keyring
        django
        webtest
        nose
+       coverage>=3.6,<3.99
        unittest2
 setenv = PYTHONPATH=../google_appengine
 
 [testenv:py26]
-commands = nosetests --ignore-files=test_oauth2client_appengine\.py
+commands = nosetests --with-coverage --cover-package=googleapiclient --nocapture --cover-erase --cover-tests --cover-branches --cover-min-percentage=85
 
 [testenv:py27]
-commands = nosetests
+commands = {[testenv:py26]commands}
```

Comment:
It's just referencing the commands in from py26 to make ensure the same syntax is used throughout.

Actually, I just tested something out locally and found a way to may it a whole lot cleaner.  I'll update the PR soon.